app/controllers/bigbluebutton_api_controller.rb

```diff
@@ -160,9 +160,9 @@ def create
     params.require(:meetingID)
 
     begin
-      server = Server.find_available
-    rescue ApplicationRedisRecord::RecordNotFound
-      raise InternalError, 'Could not find any available servers.'
```

Comment:
This specific message is adopted in server.rb, to not change the returned error message / break any tests.